skimage/restoration/deconvolution.py

```diff
@@ -356,22 +387,88 @@ def richardson_lucy(image, psf, iterations=50, clip=True):
     >>> camera = convolve2d(camera, psf, 'same')
     >>> camera += 0.1 * camera.std() * np.random.standard_normal(camera.shape)
     >>> deconvolved = restoration.richardson_lucy(camera, psf, 5)
-
+    Notes
```

Comment:
You have a CI failure here. Please, put a blank line before the note section. Actually, you need a blank line before each section. Can you fix it up everywhere please?

Reply:
fit it